build/en/index.html

```diff
@@ -0,0 +1,107 @@
+            <li> Open source <a href="https://raw.githubusercontent.com/networkx/networkx/master/LICENSE.txt">3-clause BSD license</a> </li>
+            <li> Well tested with over 90% code coverage</li>
+            <li> Additional benefits from Python include fast prototyping, easy to teach, and multi-platform</li>
+            </ul>
+
+          </div>
+        </div>
+      </div>
+
+      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
+        <div class="sphinxsidebarwrapper">
+          <div class="sidebarblock">
+          <h2>Contact</h2>
+            <ul class="title">
+              <li><a class="reference external" href="http://groups.google.com/group/networkx-discuss/"> Mailing list</a></li>
+              <li><a class="reference external" href="https://github.com/networkx/networkx/issues"> Issue tracker</a></li>
+              <li><a class="reference external" href="https://github.com/networkx/networkx"> Source</a></li> 
+            </ul>
+          </div>
+          
+          <div class="sidebarblock">
+          <h2>Releases</h2>
+          
+            <div class="tile">
+            <h4>Stable (<a href="https://networkx.org/documentation/stable/release/release_3.4.2.html">notes</a>)</h4>
+              <a href="http://pypi.python.org/pypi/networkx">3.4.2 — October 2024</a><br/>
+              <a href="https://networkx.org/documentation/stable/"><b>Documentation</b></a>
+            </div></br>
+            
+            <div class="tile">
+            <h4>Latest (<a href="https://networkx.org/documentation/latest/release/release_dev.html">notes</a>)</h4>
+              <a href="https://github.com/networkx/networkx">3.5 development</a><br/>
+              <a href="https://networkx.org/documentation/latest/"><b>Documentation</b></a>
+            </div></br>
+          
+            <div class="tile">
+            <h4><a id="other">Archive</a></h4>
+            
+              <ul id="versionList" style="display: none;">
+                  <script src="../_static/docversions.js"></script>
+                  <script type="text/javascript">
+                      insert_version_links();
+                  </script>
+              </ul>
+            
+              <script type="text/javascript">
+                  $("#other").click(function() {
+              	$("#versionList").toggle();
+              });
+              </script>
+            </div></br>
+          
+            <div class="tile">
+              <h4>Language</h4>
+              <select id="language-switcher" onchange="change_language()">
+                <option value="en" selected="">English</option>
+                <option value="es">Español</option>
+                <option value="pt">Português</option>
+              </select>
+            </div></br>
+          </div>
+        </div>
+      </div>
+      <div class="clearer"></div>
+    </div>
+
+  <div class="footer">
+    &copy;2014-2024, NetworkX developers.
```

Comment:
Just a small change(we have to update this everywhere else too-- not just in this PR)-- If we do make any major changes to the actual content on this page then how should networkx inform the translation team or indicate that the translated pages are out-of-date? or will that be automatically done by the translation bot?
```suggestion
    &copy;2014-2025, NetworkX developers.
```